tests/query/transactions/test_query_transactions.py

```diff
@@ -0,0 +1,36 @@
+import allure
+import iroha
+
+from tests import client,account_id
+from tests.helpers import generate_public_key
+
+
+@allure.label("sdk_test_id", "query_all_transactions")
+def test_query_all_transactions():
```

Comment:
Need to implement a fixture `GIVEN_registered_transaction`. So that it accurately ensures that there will be some transaction in any "new" iroha before the test.